draft-ietf-acme-acme.md

```diff
@@ -812,9 +812,9 @@ identifier (required, dictionary of string):
   : The identifier itself.
 
 status (required, string):
-: The status of this authorization.  Possible values are: "unknown", "pending",
-"processing", "valid", "invalid" and "revoked".  If this field is missing, then
-the default value is "pending".
+: The status of this authorization.  Possible values are: "pending", "processing",
```

Comment:
I think we can probably remove "processing" here too.  I originally had this idea that we would want to express whether the thing was waiting on client action or server action, but I'm not seeing any cases here where there would be ambiguity.